src/components/new-landing-page/logo-soup.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+export default function LogoSoup(): JSX.Element {
+    return (
+        <div className="logo-soup">
+            <h3 className="soup__header">Streamline your operations and reach more donors with the latest technology</h3>
+            <div className="soup__content">
+                <img src="/logo-soup-1.png" />
```

Comment:
The donorbox logo looks pixellated — can you find a better version of the logo?

Reply:
logo replaced